scalameta/contrib/src/main/scala/scala/meta/contrib/ScaladocParser.scala

```diff
@@ -0,0 +1,120 @@
+package scala.meta.contrib
+
+import DocToken._
+import scala.meta.Syntax
+import scala.meta.tokens.Token.Comment
+
+object ScaladocParser {
+
+  /**
+    * Parses a scaladoc [[Comment]].
+    */
+  def parseScaladoc(comment: Comment): Seq[DocToken] = {
+
+    val trimmedComment: String = comment.show[Syntax].trim
+
+    require(
+      requirement = trimmedComment.startsWith("/*") && trimmedComment.endsWith("*/"),
+      message = "Input comment is not a Scaladoc"
+    )
+
+    val scaladocSymbols = Seq('/', '*', ' ')
+
+    val lineSeparedDocString: Seq[String] =
+      trimmedComment
+        .split(Array('\n', '\r')) // Splits the Scaladoc in lines
```

Comment:
@olafurpg I have serious with my fastparser knowledge for this use case, it is the first time I use it and I am not expert enought for this type of parsing.

I have trying to find a partern for CodeBlocks, but I am unable to parse:

```
"""
bambaigimgqg
{{{
mgoqmgiqwmgqw
ignwrmiqmg
mgowqmgow
}}}
gmqwmgqwo
```

so I can obtain the code-block body. Any help for this?

Thank you :)